lib/enrollment_repository.rb

```diff
@@ -1,52 +1,48 @@
 require "csv"
-require "pry"
 require_relative "enrollment"
 require_relative "district"
-require 'json'
-require 'ostruct'
 
 class EnrollmentRepository
-  attr_accessor :enrollments
-
   def initialize
     @enrollments = {}
   end
 
-  def hash_pathway
```

Comment:
Deleted this extraneous hash_pathway definition to create consistency in the app between our high school/kindergarten data keys.